src/featureFlags/FlagContent.ts

```diff
@@ -0,0 +1,21 @@
+export class FlagContent {
+  constructor(public enabled: boolean, public value?: string) {}
+
+  public static fromString(content: string) {
+    try {
+      const { enabled, value } = JSON.parse(content);
+
+      return new FlagContent(enabled, value);
+    } catch (e) {
+      return FlagContent.empty();
+    }
+  }
+
+  public static empty() {
+    return new FlagContent(false, "");
+  }
+
+  public asString() {
```

Comment:
you can overwrite toString as well which is basically the same now I think

Reply:
Agree ☝🏻